src/gui/optionsdialog.cpp

```diff
@@ -684,7 +684,7 @@ void OptionsDialog::saveOptions()
     session->setAddTrackersEnabled(m_ui->checkEnableAddTrackers->isChecked());
     session->setAdditionalTrackers(m_ui->textTrackers->toPlainText());
     session->setGlobalMaxRatio(getMaxRatio());
-    session->setGlobalMaxSeedingMinutes(getMaxSeedingMinutes());
+    session->setGlobalMaxSeedingMinutes(std::chrono::minutes(getMaxSeedingMinutes()));
```

Comment:
I think it would be a lot nicer if `getMaxSeedingMinutes()` would return a duration instead. Then it wouldn't even need to have `Minutes` in its name. Same thing goes for `setGlobalMaxSeeding`, since it takes a proper type now, it doesn't need `Minutes` either